justfile

```diff
@@ -34,6 +37,7 @@ build-darwin-amd64:
     -ldflags="-s -w" \
     -ldflags="-X 'github.com/open-sauced/pizza-cli/pkg/utils.Version=${VERSION}'" \
     -ldflags="-X 'github.com/open-sauced/pizza-cli/pkg/utils.Sha=$(git rev-parse HEAD)'" \
+    -ldflags="-X 'github.com/open-sauced/pizza-cli/pkg/utils.Datetime=${DATETIME}'" \
```

Comment:
In the context of the `build-darwin-amd64` command, you're missing `DATETIME`. You need to add it for this script explicitly above:

```justfile
build-darwin-amd64:
  #!/usr/bin/env sh

  echo "Building darwin amd64"

  export VERSION="${RELEASE_TAG_VERSION:-dev}"
  export DATETIME=$(date -u +"%Y-%m-%d %H:%M:%S")
  export CGO_ENABLED=0
  export GOOS="darwin"
  export GOARCH="amd64"

  go build \
    -ldflags="-s -w" \
    -ldflags="-X 'github.com/open-sauced/pizza-cli/pkg/utils.Version=${VERSION}'" \
    -ldflags="-X 'github.com/open-sauced/pizza-cli/pkg/utils.Sha=$(git rev-parse HEAD)'" \
    -ldflags="-X 'github.com/open-sauced/pizza-cli/pkg/utils.Datetime=${DATETIME}'" \
    -o build/pizza-${GOOS}-${GOARCH}
```

Reply:
Same goes for others below in other commands: `just` is alot like `make` in this where each command is its own thing (like it's own script) run in it's own shell interpreter.

Reply:
Maybe I'll refactor this file to make it abit more clear what's going on once this lands 🚀

Reply:
Go ahead and add

```sh
  export DATETIME=$(date -u +"%Y-%m-%d %H:%M:%S")
```

to all the other build scripts here and we can merge this in 👍🏼

Reply:
Not sure why I didn't see that. When I ran `just build-all` everything went smooth. Anyway, will check where I missed it in the other build scripts.